configserver/src/main/sh/start-configserver

```diff
@@ -156,6 +156,10 @@ export LD_PRELOAD=${VESPA_HOME}/lib64/vespa/malloc/libvespamalloc.so
 printenv > $cfpfile
 fixddir $bundlecachedir
 
+# Delete mdbm lock files
+# TODO: Remove when scoreboard is no longer used
+$(ROOT)/bin64/mdbm_delete_lockfiles $(ROOT)/var/scoreboards/configserver.sbm || true
```

Comment:
Is mdbm* available on all distributions?

Reply:
No, but we handle that the command fails. 

This cleanup is not necessary on all distributions, but I am not sure how we can check where this should be done, at least not when it is done in a start script.

Reply:
Don't we have a yahoo specific startfile ?
We could also do it inside the container.
This should also be done on all container nodes. There has also been tenant issues related to this.